extension/ui/components/conversation/UserMessage.tsx

```diff
@@ -33,12 +37,13 @@ export function UserMessage({
     () => ({
       sup: CiteBlock,
       mention: MentionBlock,
+      url_mention: UrlMentionBlock,
```

Comment:
My only worry is that we may incentifize the model to use this instead of the regular markdown format for later message. Can we use a slightly more explciit name such as `content_node_mention` :pray:

Reply:
Fair point, I'm also a tiny bit worried about having the full url there. Don't you think the model might want to browse it?

Reply:
Welll it's user provided so I think it's fine to have the URL